EasyPost/Services/EventService.cs

```diff
@@ -42,6 +43,23 @@ internal EventService(EasyPostClient client)
         [CrudOperations.Read]
         public async Task<Event> Retrieve(string id) => await Get<Event>($"events/{id}");
 
+        /// <summary>
+        ///     Retrieve all <see cref="Payload"/>s for an <see cref="Event"/>.
+        /// </summary>
+        /// <param name="event"><see cref="Event"/> to retrieve payloads for.</param>
+        /// <returns>A list of <see cref="Payload"/> objects.</returns>
+        public async Task<List<Payload>> RetrieveAllPayloadsForEvent(Event @event) => await Get<List<Payload>>($"events/{@event.Id}/payloads", rootElement: "payloads");
```

Comment:
This function should be called `RetrieveAllPayloads` to match other CLs

Reply:
Gross, I thought we decided for the hybrid design we would be passing whole objects around in function, not just the object IDs.

But I can't find anything to support that (C# was rewritten not with the hybrid design, and Java and PHP that were rewritten with the hybrid design are passing in only the ID of the object its executing against).

I hate that, but I'll adhere to it to make it consistent.